website/content/docs/agent/autoauth/index.mdx

```diff
@@ -183,3 +185,29 @@ These configuration values are common to all Sinks:
 
 - `config` `(object: required)` - Configuration of the sink itself. See the
   sidebar for information about each sink.
+
+### Auto Auth Example
+
+```python
```

Comment:
hcl doesn't work here? (The rendered preview looks fine, was just surprised.)

Reply:
Yeah it probably would work. I'll get these updated.